public/keymaps/blackplum_default.json

```diff
@@ -0,0 +1 @@
+{"keyboard":"blackplum","keymap":"default","layout":"LAYOUT_68_ansi","layers":[["KC_ESC","KC_1","KC_2","KC_3","KC_4","KC_5","KC_6","KC_7","KC_8","KC_9","KC_0","KC_MINS","KC_EQL","KC_BSPC","KC_INS","KC_HOME","KC_TAB","KC_Q","KC_W","KC_E","KC_R","KC_T","KC_Y","KC_U","KC_I","KC_O","KC_P","KC_LBRC","KC_RBRC","KC_BSLS","KC_DEL","KC_END","KC_CAPS","KC_A","KC_S","KC_D","KC_F","KC_G","KC_H","KC_J","KC_K","KC_L","KC_SCLN","KC_QUOT","KC_ENT","KC_LSFT","KC_Z","KC_X","KC_C","KC_V","KC_B","KC_N","KC_N","KC_COMM","KC_DOT","KC_SLSH","KC_RSFT","KC_UP","KC_LCTL","KC_LGUI","KC_LALT","KC_SPC","KC_RALT","MO(1)","KC_RCTL","KC_LEFT","KC_DOWN","KC_RGHT"],["KC_GRV","KC_F1","KC_F2","KC_F3","KC_F4","KC_F5","KC_F6","KC_F7","KC_F8","KC_F9","KC_F10","KC_F11","KC_F12","KC_TRNS","KC_MPLY","KC_HOME","KC_TRNS","RGB_MOD","RGB_HUI","RGB_SAI","RGB_VAI","KC_TRNS","KC_TRNS","KC_TRNS","KC_TRNS","KC_TRNS","KC_PSCR","KC_SLCK","KC_PAUS","KC_TRNS","KC_MUTE","KC_END","KC_TRNS","RGB_TOG","RGB_HUD","RGB_SAD","RGB_VAD","KC_TRNS","KC_TRNS","KC_TRNS","KC_TRNS","KC_TRNS","KC_TRNS","KC_TRNS","KC_TRNS","KC_TRNS","KC_TRNS","KC_TRNS","KC_TRNS","KC_TRNS","KC_TRNS","KC_TRNS","KC_TRNS","KC_TRNS","KC_TRNS","KC_TRNS","KC_TRNS","KC__VOLUP","KC_TRNS","TG_GUI","KC_TRNS","RESET","KC_TRNS","KC_TRNS","KC_TRNS","KC_MRWD","KC__VOLDOWN","KC_MFFD"]],"author":"","notes":""}
```

Comment:
## Changes in this suggestion:

- `keymap` -> `default_e264f01`,
  - `keymap` can't actually be set to `default` here because Configurator's API works from a clone of qmk_firmware, and attempts to create `keyboards/<keyboard>/keymaps/<keymap>/keymap.c` at compile time, but this fails if that file already exists
- new `commit` key
  - I periodically go back from time to time to check if the keymaps have been updated on qmk_firmware. Basically, it's just internal tracking.
- Keycodes changed to match/resemble default keymap in qmk_firmware repo:
  - Layer 0:
    - `KC_ESC` -> `KC_GESC`
    - `KC_HOME` -> `KC_PGUP`
    - `KC_END` -> `KC_PGDN`
    - `KC_N` -> `KC_M`
  - Layer 1:
    - `KC__VOLUP` -> `KC_VOLU`
    - `KC_MRWD` -> `KC_MPRV`
    - `KC__VOLDOWN` -> `KC_VOLD`
    - `KC_MFFD` -> `KC_MNXT`


```suggestion
{"keyboard":"blackplum","keymap":"default_e264f01","commit":"e264f0151da0025dbb53dfc5a70bce13105a04d8","layout":"LAYOUT_68_ansi","layers":[["KC_GESC","KC_1","KC_2","KC_3","KC_4","KC_5","KC_6","KC_7","KC_8","KC_9","KC_0","KC_MINS","KC_EQL","KC_BSPC","KC_INS","KC_PGUP","KC_TAB","KC_Q","KC_W","KC_E","KC_R","KC_T","KC_Y","KC_U","KC_I","KC_O","KC_P","KC_LBRC","KC_RBRC","KC_BSLS","KC_DEL","KC_PGDN","KC_CAPS","KC_A","KC_S","KC_D","KC_F","KC_G","KC_H","KC_J","KC_K","KC_L","KC_SCLN","KC_QUOT","KC_ENT","KC_LSFT","KC_Z","KC_X","KC_C","KC_V","KC_B","KC_N","KC_M","KC_COMM","KC_DOT","KC_SLSH","KC_RSFT","KC_UP","KC_LCTL","KC_LGUI","KC_LALT","KC_SPC","KC_RALT","MO(1)","KC_RCTL","KC_LEFT","KC_DOWN","KC_RGHT"],["KC_GRV","KC_F1","KC_F2","KC_F3","KC_F4","KC_F5","KC_F6","KC_F7","KC_F8","KC_F9","KC_F10","KC_F11","KC_F12","KC_TRNS","KC_MPLY","KC_HOME","KC_TRNS","RGB_MOD","RGB_HUI","RGB_SAI","RGB_VAI","KC_TRNS","KC_TRNS","KC_TRNS","KC_TRNS","KC_TRNS","KC_PSCR","KC_SLCK","KC_PAUS","KC_TRNS","KC_MUTE","KC_END","KC_TRNS","RGB_TOG","RGB_HUD","RGB_SAD","RGB_VAD","KC_TRNS","KC_TRNS","KC_TRNS","KC_TRNS","KC_TRNS","KC_TRNS","KC_TRNS","KC_TRNS","KC_TRNS","KC_TRNS","KC_TRNS","KC_TRNS","KC_TRNS","KC_TRNS","KC_TRNS","KC_TRNS","KC_TRNS","KC_TRNS","KC_TRNS","KC_TRNS","KC_VOLU","KC_TRNS","TG_GUI","KC_TRNS","RESET","KC_TRNS","KC_TRNS","KC_TRNS","KC_MPRV","KC_VOLD","KC_MNXT"]]}
```

Reply:
thanks, I have updated.